tests/phpunit/CRM/Event/Form/Registration/ConfirmTest.php

```diff
@@ -584,4 +584,89 @@ public function testNoteSubmission() {
     list($contact_id, $participant_id) = $this->submitWithNote($event, $contact_id);
   }
 
+  /**
+   * Test for Multiple participant.
+   */
+  public function testMultipleParticipant() {
+    $params = ['is_monetary' => 1, 'financial_type_id' => 1];
+    $event = $this->eventCreate($params);
+    CRM_Event_Form_Registration_Confirm::testSubmit([
+      'id' => $event['id'],
+      'contributeMode' => 'direct',
+      'registerByID' => $this->createLoggedInUser(),
+      'totalAmount' => 440,
+      'event' => reset($event['values']),
+      'params' => [
+        [
+          'qfKey' => 'e6eb2903eae63d4c5c6cc70bfdda8741_2801',
+          'entryURL' => "http://dmaster.local/civicrm/event/register?reset=1&amp;id={$event['id']}",
+          'first_name' => 'Participant1',
+          'last_name' => 'LastName',
+          'email-Primary' => 'participant1@example.com',
+          'additional_participants' => 2,
+          'payment_processor_id' => 0,
+          'bypass_payment' => '',
+          'MAX_FILE_SIZE' => '33554432',
+          'is_primary' => 1,
+          'is_pay_later' => 1,
+          'campaign_id' => NULL,
+          'defaultRole' => 1,
+          'participant_role_id' => '1',
+          'currencyID' => 'USD',
+          'amount_level' => 'Tiny-tots (ages 5-8) - 1',
+          'amount' => '100.00',
+          'tax_amount' => 10,
+          'ip_address' => '127.0.0.1',
+          'invoiceID' => '57adc34957a29171948e8643ce906332',
+          'trxn_id' => '123456789',
+          'button' => '_qf_Register_upload',
+        ],
+        [
+          'qfKey' => 'e6eb2903eae63d4c5c6cc70bfdda8741_2801',
+          'entryURL' => "http://dmaster.local/civicrm/event/register?reset=1&amp;id={$event['id']}",
+          'first_name' => 'Participant2',
+          'last_name' => 'LastName',
+          'email-Primary' => 'participant2@example.com',
+          'scriptFee' => '',
+          'scriptArray' => '',
+          'campaign_id' => NULL,
+          'is_pay_later' => 1,
+          'participant_role_id' => '1',
+          'currencyID' => 'USD',
+          'amount_level' => 'Tiny-tots (ages 9-18) - 1',
+          'amount' => '200.00',
+          'tax_amount' => 20,
+        ],
+        [
+          'qfKey' => 'e6eb2903eae63d4c5c6cc70bfdda8741_2801',
+          'entryURL' => "http://dmaster.local/civicrm/event/register?reset=1&amp;id={$event['id']}",
+          'first_name' => 'Participant3',
+          'last_name' => 'LastName',
+          'email-Primary' => 'participant3@example.com',
+          'scriptFee' => '',
```

Comment:
it's not a biggie but I've started taking out scriptFee & scriptArray from other tests as I've dug & concluded they should go entirely & are not used

Reply:
Removed - see https://lab.civicrm.org/dev/event/issues/19